src/reference/interactive/index.pug

```diff
@@ -66,15 +68,15 @@ block reference
         li Control who can access your project
         li Publish your project for everyone on Beam to see and use
 
-    p When a #[a(href='#the-gameclient') Game Client] connects to the Interactive Service, the game client provides your project's ID. The service reads this ID and sets up an interactive session with the saved settings and controls created in the studio.
+    p When a #[a(href='#the-gameclient') Game Client] connects to the Interactive Service, it provides your project's ID to the service. The service reads this ID and sets up an interactive session with the saved settings and controls created in the Studio.
 
     h3#participants Participants
 
-    p Participants are Beam users that are watching a broadcast in a broadcaster's channel. When they join the channel they are connected to the interactive session. They are provided with a set of on-screen controls that they can use to affect the broadcast. A #[a(href='#the-gameclient') Game Client] can react to game scenarios and interactive input to change the controls displayed to participants.
+    p Participants are viewers of a live stream. They are Beam users watching a broadcast in a broadcaster's channel. When they join the channel, they are connected to the interactive session. They are provided with a set of on-screen UI controls that they can use to affect the broadcast. A #[a(href='#the-gameclient') Game Client] can react to game scenarios and interactive input to change the controls displayed to participants.
 
     h2#interactive-experience-structure Interactive Experience Structure
 
-    p An Interactive experience contains a hierarchical structure of various elements that interact with and affect the experience.
+    p In this section, let’s look at the hierarchical structure of the various elements that developers can use to create this interaction.
```

Comment:
The sudden change to a second person directive tone is jarring here.

This section is devoid of "person" usually.